packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -116,12 +117,29 @@ export class ActiveWorkflowRunner {
 			throw new ResponseHelper.ResponseError('The "activeWorkflows" instance did not get initialized yet.', 404, 404);
 		}
 
-		const webhook = await Db.collections.Webhook?.findOne({ webhookPath: path, method: httpMethod }) as IWebhookDb;
+		let webhook = await Db.collections.Webhook?.findOne({ webhookPath: path, method: httpMethod }) as IWebhookDb;
+		let webhookId: string | undefined;
 
-		// check if something exist
+		// check if path is dynamic
 		if (webhook === undefined) {
-			// The requested webhook is not registered
-			throw new ResponseHelper.ResponseError(`The requested webhook "${httpMethod} ${path}" is not registered.`, 404, 404);
+			// check if a dynamic webhook path exists
+			const pathElements = path.split('/');
+			webhookId = pathElements.shift();
```

Comment:
All comments in this blog also apply for TestWebhooks.ts.
There should be no need to check in the database if `pathElements.length === 0` or if > 0 and `webhookData.path` does not contain any `:` as either of it means that it is not a dynamic path.

Reply:
absolutely, gonna add them to `TestWebhooks.ts`. Unfortunately I don't quite understand the second part of the comment. `pathElements > 0` is given, in the database the `webhook.path` contains `:`, the actual path that we are receiving does not, so we split the path and remove the first element (the webhookId) so we then have the same length of the db path and the path that was called with the params